src/lava/magma/core/model/py/ports.py

```diff
@@ -160,12 +280,51 @@ def send(self, data: int, idx: int):
 
 
 class VarPortCmd:
+    """Helper class to specify constants."""
     GET = enum_to_np(0)
     SET = enum_to_np(1)
 
 
 class PyRefPort(AbstractPyPort):
-    """Python implementation of RefPort used within AbstractPyProcessModels."""
+    """Python implementation of RefPort used within AbstractPyProcessModels.
```

Comment:
What's a RefPort?

Reply:
Added explanation of RefPorts, what they do and how they work.